src/js/components/user-management/userform.js

```diff
@@ -88,19 +90,21 @@ export default class UserForm extends React.Component {
             submitLabel={isCreation ? 'Create user' : 'Save changes'}
             submitButtonId="submit_button"
             showButtons={true}
+            autocomplete="off"
           >
-            <TextInput hint="Email" label="Email" id="email" value={user.email} validations="isLength:1,isEmail" required={isCreation} />
+            <TextInput hint="Email" label="Email" id="email" value={user.email} validations="isLength:1,isEmail" required={isCreation} autocomplete="off" />
             {editPass ? (
               <PasswordInput
                 className="edit-pass margin-top-small"
-                id="password"
+                id="password_new"
```

Comment:
this is intended, because if we leave the id as password, the lastpass chrome/firefox plugin will start autocompleting it. I think the validation for length < 2 is useless, as we are using a custom field which verifies the password strength. Is it correct?

Reply:
yep I think the validation as it is doesn't provide a real benefit, however to deactivate the autocomplete _I think_ setting the value for autocomplete to `autocomplete="new-password"` should be helpful as the input type is set to password by in the password input component... but you might have tried that already I guess, in that case it's ok as you changed it here 👌

Reply:
yup, tried a lot of different combinations of autocomplete on the fields but none of them blocked the lastpass plugin. The only combination was autocomplete off at the form level, combined with a different id for the password field.